tq/transfer.go

```diff
@@ -34,17 +34,25 @@ type Transfer struct {
 	Size          int64        `json:"size"`
 	Authenticated bool         `json:"authenticated,omitempty"`
 	Actions       ActionSet    `json:"actions,omitempty"`
+	Links         ActionSet    `json:"_links,omitempty"`
 	Error         *ObjectError `json:"error,omitempty"`
 	Path          string       `json:"path,omitempty"`
 }
 
-func (t *Transfer) Rel(name string) (*Action, bool) {
-	if t.Actions == nil {
-		return nil, false
+func (t *Transfer) Rel(name string) (*Action, error) {
+	a, err := t.Actions.Get(name)
+	if a != nil || err != nil {
+		return a, err
 	}
 
-	rel, ok := t.Actions[name]
-	return rel, ok
+	if t.Links != nil {
+		a, err := t.Links.Get(name)
+		if a != nil || err != nil {
+			return a, err
+		}
+	}
+
+	return nil, nil
```

Comment:
Sure, it works. I just like early returns in Go code to reduce extra nesting elsewhere in the function. I can't guarantee I won't point out when you can use early returns to reduce extra nesting the next time this comes up, or that I won't make this change myself the next time I have to change this function (which isn't often of course).